storage/localstore/localstore_test.go

```diff
@@ -585,8 +596,8 @@ func TestDBDebugIndexes(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// for reference: testIndexCounts(t *testing.T, pushIndex, pullIndex, gcIndex, gcExcludeIndex, pinIndex, retrievalDataIndex, retrievalAccessIndex int, indexInfo map[string]int)
-	testIndexCounts(t, 1, 1, 0, 0, 0, 1, 0, indexCounts)
+	// for reference: testIndexCounts(t *testing.T, pushIndex, pullIndex, gcIndex, gcExcludeIndex, pinIndex, data, metaIndex int, indexInfo map[string]int)
+	testIndexCounts(t, 1, 1, 0, 0, 0, 1, 1, indexCounts)
```

Comment:
comment on the discrepancy with the old test?

Reply:
Access index is replaced with meta index which contains store timestamp. Access index did not have an entry, but meta index now has. Test helper testIndexCounts may be better constructed so that it is obvious which indexes are tested. I do not think that this function is designed nicely.